js/privatebin.js

```diff
@@ -362,6 +361,36 @@ jQuery.PrivateBin = (function($, RawDeflate) {
             }
         };
 
+        /**
+         * converts size in bytes to human readable string
+         *
+         * The string is expected to be optional digits, followed by a time.
+         * Supported times are: min, hour, day, month, year, never
+         * Examples: 5min, 13hour, never
+         *
+         * @name Helper.humanizeBytes
+         * @function
+         * @param  {number} bytes
+         * @param {decimals} decimals - optional
+         * @return {String}
+         */
+        me.humanizeBytes = function(bytes, decimals = 2) {
+            const sizes = [
+                I18n._('Bytes'), I18n._('KB'), I18n._('MB'), I18n._('GB'),
+                I18n._('TB'), I18n._('PB'), I18n._('EB'), I18n._('ZB'), I18n._('YB')
+            ];
+            const signifierByte = I18n._('Byte');
+            if (bytes === 0) return me.sprintf('0 %s', sizes[0]);
+            if (bytes === 1) return me.sprintf('1 %s', signifierByte);
```

Comment:
If I understand the intent behind these two lines correctly, you want to translate "Byte" differently, if it applies to 0 instead of 1. I would recommend to let the translation systems [plural form support](https://github.com/PrivateBin/PrivateBin/wiki/Translation#plural-forms) handle this, as in different languages different rules apply for the use of a plural form for 0 or other numbers - in English and German we do need to use the plural for 0 of something, but for example in French or Chinese this not the case:
https://github.com/PrivateBin/PrivateBin/blob/33bcce593a3e8baa7ebdf6abf00175b399582fc9/js/privatebin.js#L781-L784

So my suggestion would be to instead have a line like (the translate method supports sprintf like syntax as well):
```javascript
            if (bytes < 1024) return I18n._('%d Byte', bytes);
```